src/components/Form.js

```diff
@@ -0,0 +1,39 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+
+import StyledForm from '../style/StyledForm';
+
+import {Workouts} from './db';
+
+export default function Form() {
+  const [plans, setPlans] = useState([]);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  function handleChange(event) {
+    const newExercise = event.target.value;
+    setPlans([...plans, newExercise]);
+  }
+
+  const Exercise = Workouts[0].exercise;
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <label htmlFor="input1">Exercise:</label>
+      <select value={Exercise.value} onChange={handleChange}>
+        {Exercise.map(workout => (
+          <option key={nanoid()}>{workout}</option>
```

Comment:
```suggestion
        {exercises.map(exercise => (
          <option key={nanoid()}>{exercise}</option>
```
As mentioned above: If you rename everything to `exercises` you can create a clear naming convention for this `map()` method with `exercises` and `exercise`. Also because `workout` is related something else!?